tests/run.sh

```diff
@@ -28,17 +28,18 @@ clone() {
 
 mkdir -p $TEST_MINIMAL_DIR
 clone $GITHUB_PLENARY "$TEST_MINIMAL_DIR/plenary.nvim"
-nvim --headless -u tests/environments/minimal.lua -c "PlenaryBustedDirectory tests/ {minimal_init = 'tests/environments/minimal.lua', sequential = true}"
```

Comment:
Scope the tests to use specific folders, in this case `conversion` and `plugin`, to avoid testing on the telescope spec when the environment does not have Telescope installed

I guess this should be a good start while we figure out the right design